db/migrate/20220407130736_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.references :author, null: false, foreign_key:{ to_table: :users }
+      t.references :posts, null: false, foreign_key: { to_table: :posts }
```

Comment:
- [ ] Please note that just like the `:author` reference this line should reference a `:post` not a `:posts`. 
- [ ] [OPTIONAL] in this case you don't need to specify the `to_table` parameter. rails can figure out this relationship based on the reference name.

```suggestion
      t.references :post, null: false, foreign_key: true
```